src/test/java/org/apache/commons/lang3/Supplementary.java

```diff
@@ -25,25 +25,24 @@
 public class Supplementary {
 
     /**
-     * Supplementary character U+20000 See https://www.oracle.com/technical-resources/articles/javase/supplementary.html
+     * Incomplete supplementary character U+20000, low surrogate only. See
+     * https://www.oracle.com/technical-resources/articles/javase/supplementary.html
      */
-    static final String CharU20000 = "\uD840\uDC00";
+    static final String CharU20000SupplCharLow = "\uDC00";
```

Comment:
These changes duplicate your changes in your other PR. Rebase on git master and don't make these changes here.